docker-for-windows/wsl.md

```diff
@@ -77,6 +77,8 @@ Ensure you have completed the steps described in the Prerequisites section **bef
 
     Optionally, select any additional distributions you would like to enable WSL 2 on.
 
+    Please note that the WSL Integrations bits running in your distro depend on glibc. This make it difficult to run it on musl-based distros such as Alpine Linux.
+
```

Comment:
Another comment about this section while you're editing it. We talk on lines 74 and 80 about "WSL integration" and on line 78 about "enabling WSL 2". Isn't it really "Docker integration" and "enabling Docker" in that distro? I think that would be clearer.

Reply:
For now we distinguish:
- Enabling the WSL 2 backend (running docker within WSL 2)
- Enabling the WSL integration (which is running the proxy to enable the docker experience within any of the user's WSL distros).

Please note that in the docs we refer to terms present in the UI. If we want to change that (and I agree with you, the vocabulary is confusing, and was obviously chosen by a french software engineer rather than by a native english PM :D), we need to do it both in UI (settings screen) and docs in sync.

Reply:
Well, I only just thought of it too. I think we should call it the "Docker-WSL integration" (in both UI and docs): then it's clear from both directions. I'll make a ticket. cc @usha-mandya

Reply:
Agree. We should clarify that it is the Docker integration within the WSL distro. Not sure if we should hold this PR up until the changes are in.

Reply:
Agreed, we can do it separately from this PR.